packager/media/formats/mp2t/pes_packet_generator.cc

```diff
@@ -52,15 +52,17 @@ bool PesPacketGenerator::Initialize(const StreamInfo& stream_info) {
     converter_.reset(new NalUnitToByteStreamConverter());
     return converter_->Initialize(video_stream_info.codec_config().data(),
                                   video_stream_info.codec_config().size());
-  } else if (stream_type_ == kStreamAudio) {
+  }
+  if (stream_type_ == kStreamAudio) {
     const AudioStreamInfo& audio_stream_info =
         static_cast<const AudioStreamInfo&>(stream_info);
     timescale_scale_ = kTsTimescale / audio_stream_info.time_scale();
     if (audio_stream_info.codec() == Codec::kCodecAAC) {
       audio_stream_id_ = kAacAudioStreamId;
       adts_converter_.reset(new AACAudioSpecificConfig());
       return adts_converter_->Parse(audio_stream_info.codec_config());
-    } else if (audio_stream_info.codec() == Codec::kCodecAC3 ||
+    }
+    if (audio_stream_info.codec() == Codec::kCodecAC3 ||
```

Comment:
Can you align |audio_stream_info| in the next line with |audio_stream_info| in this line?